bridge/github/import.go

```diff
@@ -15,94 +15,155 @@ import (
 	"github.com/MichaelMure/git-bug/util/text"
 )
 
+const EMPTY_TITLE_PLACEHOLDER = "<empty string>"
+
 // githubImporter implement the Importer interface
 type githubImporter struct {
 	conf core.Configuration
 
-	// default client
-	client *githubv4.Client
-
-	// iterator
-	iterator *iterator
+	// mediator to access the Github API
+	mediator *importMediator
 
 	// send only channel
 	out chan<- core.ImportResult
 }
 
-func (gi *githubImporter) Init(_ context.Context, repo *cache.RepoCache, conf core.Configuration) error {
+func (gi *githubImporter) Init(_ context.Context, _ *cache.RepoCache, conf core.Configuration) error {
 	gi.conf = conf
+	return nil
```

Comment:
So you moved the credential loading + client creation in `ImportAll`. Not what I imagined but I guess that make sense.
My tiny annoyance with that though is that this has now a different behavior than the other bridges. Could you make the same change there as well?

Reply:
My original intention was to simplify the code. Looking at it now makes me also wonder if it really makes any significant difference. The difference in the local code seems to be: When doing the credential loading + client creation in `ImportAll`, then we do not have to save a `githubv4.Client` variable in the `githubImporter` struct, and there is no risk in having two non-identical `*cache.RepoCache` parameters in the calls to `Init` and `ImportAll`.

I could undo this change and move the credential loading + client creation back into `Init` if that's what you prefer. Please let me know again what you prefer: (1) move credential loading + client creation back into `Init`, or (2) keeping it in `ImportAll` and make the same change in all the other bridges.

Reply:
My original intention was to simplify the code. Looking at it now makes me also wonder if it really makes any significant difference. The difference in the local code seems to be: When doing the credential loading + client creation in `ImportAll`, then we do not have to save a `githubv4.Client` variable in the `githubImporter` struct, and there is no risk in having two non-identical `*cache.RepoCache` parameters in the calls to `Init` and `ImportAll`.

I could undo this change and move the credential loading + client creation back into `Init` if that's what you prefer. Please let me know again what you prefer: (1) move credential loading + client creation back into `Init`, or (2) keeping it in `ImportAll` and make the same change in all the other bridges.

Reply:
I did that with 3d14e2e for the Gitlab and Jira bridges. There was no need to change anything in the Launchpad bridge.